### DISCUSSION TITLE
Performance issues and display glitches

### DISCUSSION THREAD

**Discussion by dhbrojas** [community/community#112193] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Description

Hey there, I've been having a tough time with GitHub's website for about a year now. It's gotten to the point where I just clone everything locally or use other tools instead because the website just isn't playing nice.

- Often, when switching between pages, the whole layout momentarily breaks for a sec.
- The file explorer feels like it's moving through molasses. Just opening a folder or switching files takes a few seconds.
- Scrolling through code sometimes makes parts of the UI vanish.

Overall, my main pain point is with the "snappiness" of the UI. I wish it were faster and more responsive.

I say this with the utmost love and respect for the developers. I understand it must be quite a challenge managing this big a site, I'm just sharing my experience and hope it's of value to you ❤️

### Environment

Safari Version 16.1 (18614.2.9.1.12) on Apple M1 Pro 16GB running macOS Version 13.0.1 (22A400)

**Comment by rvt** [comment]
Same here on Safari.. Sometimes I use chrome just to quickly read something, but it's utterly annoying to open a non standard browser just to few a few lines of text.
